composer.json

```diff
@@ -46,8 +46,8 @@
 		"composer/installers": "~1.0"
 	},
 	"scripts": {
-		"format": "phpcbf --standard=phpcs.xml.dist --report-summary --report-source",
-		"lint": "phpcs --standard=phpcs.xml.dist",
+		"format": "bash ./scripts/format-php.sh",
+		"lint": "bash ./scripts/lint-php.sh",
```

Comment:
In general I think we use the `bin` folder for scripts.

Reply:
Thanks! Updated in https://github.com/WordPress/gutenberg/pull/61504/commits/8e1b30d954f1045c02dab34848b5f6dfbae67af2